README.md

```diff
@@ -143,12 +162,11 @@ export AWS_SECRET_ACCESS_KEY=<aws_secret_key>
 
 ## Deployment
 
-There are essentially 2 main modes of deployment:
+There are two main modes of deployment:
 
-* Solace PubSub+ Console Connected: The Event Management Agent connects to the event management region and can be
-  controlled remotely via Event Portal
+* Connected: The Event Management Agent connects to the Solace PubSub+ Event Portal.
```

Comment:
Should indicate what connected means.
- For connected: Scans must be initiated from Event Portal. Completed scan results will be uploaded to Event Portal automatically.

Reply:
Done